python/tvm/relay/ad.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=no-else-return
+# pylint: disable=unidiomatic-typecheck
+"""
+The set of automatic differentiation algorithms in Relay.
+"""
+from . import _ad
+
+def fo_with_gradient(expr, mod=None):
```

Comment:
Let us consider rename ad-> gradient, as it is more comment name among frameworks